### PR TITLE
Issue 130 add windows and mac testing

### LINKED ISSUE #130: Windows and MacOS support
Check whether we can get everything working on Windows and MacOS, and add these to travis

### PR DESCRIPTION
**Pull request by martinjrobins**
# Description

Fixes #130 

Adds windows testing via appveyor and mac os x testing via travis. Both only run the unit and integration tests, without scikits.odes installed (might need a separate issue for installing this on windows/mac)

Note: I have had to skip a *lot* of tests in the case where scikits.odes are not installed. Once we have Ivan's timestepper in pybamm, might want to make this the default solver when scikits.odes are not installed

## Type of change

Please delete options that are not relevant.

- [x] New feature (non-breaking change which adds functionality)

# Key checklist:

- [x] No style issues: `$ flake8`
- [x] All tests pass: `$ python run-tests.py --unit`
- [x] The documentation builds: `$ cd docs` and then `$ make clean; make html`

You can run all three at once, using `$ python run-tests.py --quick`.

## Further checks: 

- [x] Code is commented, particularly in hard-to-understand areas
- [x] Tests added that prove fix is effective or that feature works
- [x] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: .appveyor.yml [hunk 1]
```diff
@@ -0,0 +1,20 @@
+branches:
+  only:
+    - issue-130-windows
```
Comment (tinosulzer) [inline .appveyor.yml]: Should this be removed before merge?

File: setup.py [hunk 2]
```diff
@@ -14,6 +14,7 @@
     author="Valentin Sulzer",
```
Comment (tinosulzer) [inline setup.py]: we should definitely change this now!

File: .appveyor.yml (hunk 1 above)
Comment (martinjrobins) [reply .appveyor.yml]: Yes! good spot

File: setup.py (hunk 2 above)
Comment (martinjrobins) [reply setup.py]: Perhaps we can say "The PyBaMM development team", and have an AUTHORS file with all the names?

Comment (tinosulzer) [reply setup.py]: yep sounds good, we could even just leave `author` out completely

**Comment by martinjrobins** [comment]
> so we could just make that the default solver for all ODE models? 

I've gone through and done this for all the ode models I could find

File: setup.py (hunk 2 above)
Comment (martinjrobins) [reply setup.py]: ok, I've now removed the author and author email

**Pull request merged**